### PR TITLE
font size headings

### PR DESCRIPTION
**Pull request by Ian-Hitt**

### CODE REVIEW

**Comment by mikecreuzer** [comment]
Before merging this, we might want to discuss leaving this alone until we can have a proper discussion on it. Something we likely need to decide on a direction rather than screw up every component we have right now.

**Comment by Ian-Hitt** [comment]
We already had a long team discussion on it and that's what we chose to do. It's just reverting to the way it was before Grant added vw scaling

File: src/Alert/Alert.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const Alert = ({ status = 'info', variant = 'left-accent', ...rest }: Ale
 
 export const AlertTitle = (props: BoxProps) => <Heading kind="h6" {...props} />;
 
-export const AlertDescription = (props: BoxProps) => <Text {...props} />;
+export const AlertDescription = (props: BoxProps) => <Text color="inherit" {...props} />;
```
Comment (tushar-singh) [inline src/Alert/Alert.tsx:32]: wouldn't this be the default anyway? or is Text setting its own color?

File: src/Alert/examples.tsx [hunk 2]
```diff
@@ -59,3 +60,16 @@ stories.add('Solid', () => {
...
 });
+
+stories.add('Opacity', () => {
+    return (
+        <Alert status="error" variant="opacity" justifyContent="center">
+            <AlertIcon />
+            <AlertTitle display="inline-block" mr="spacing-sm">
+                Your browser is outdated!
+            </AlertTitle>
+            <AlertDescription display="inline-block">Your Chakra experience may be degraded.</AlertDescription>
```
Comment (tushar-singh) [inline src/Alert/examples.tsx:71]: chakra lol

**Review by tushar-singh — APPROVED** [review]

**Pull request merged**